### ISSUE TITLE
Tiger VNC Menu pops up with CMD+F8, Control+F8 and opt+F8

### ISSUE THREAD

**Issue by prasskaly** [TigerVNC/tigervnc#1422]
I would like to map CMD+F8, Control+F8 in my VNC Linux session. 
But, I see that Tiger VNC options' menu is triggered in all cases (although only F8 is configured as the Menu Key).

Is it possible to free up these key combinations ?

**Comment by CendioOssman** [comment]
Seems reasonable. Our intention was always that you press just that button to open the menu, not combinations.

Until this is fixed you can change which button opens the menu. So try to pick something that doesn't conflict with your applications.

**Comment by samhed** [comment]
I guess this won't be a problem after #1375?

**Comment by CendioOssman** [comment]
That's correct. However that is unclear when it will be merged, so this could be relevant until then.

**Comment by prasskaly** [comment]
Hi, 
This doesn't seem fixed with the latest available TigerVNC on Mac v1.15.0. The only way I got it working is to map the menu key to None. 
Any other key set triggers the option menu with most of the modifiers (Ctrl, Cmd, Esc, Shift, Opt).
